docs/README.md

```diff
@@ -5,4 +5,26 @@ The posts in this folder are intended for sharing technical learnings from some
 * [Practical guide on how to scure an Azure Machine Learning project](azureml-security.md)
 * [How to run and debug Azure Machine Learning projects on local dev machine](local-development.md)
 * [How to manage the different types of variables used in ML projects](variable-management.md)
-* How to manage data for ML projects - coming soon
\ No newline at end of file
+* How to manage data for ML projects - coming soon
+
+# Folder Structure
+
+```bash
+├─ architecture # Azure ML architecture information 
+├─ media # contains images for docs
+├─ process # enginnering fundamentals 
+```
+
+## Architecture
+
+This folder contains Azure Machine Learning Architecture related information.
+
+- [Azure Machine Learning Environment: How to use custom software and depencies in Azure ML pipeline](architecture/AzureMLEnvironment.md)
```

Comment:
This is a really good document, clear to the point. But I wonder if we should leave it in the root folder or move the existing files in the root folder to the architecture folder, because all the existing articles in the root folder are architecture related.

Reply:
i will move all docs in root into sub folder